daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/ledgerinteraction/JsonLedgerClient.scala

```diff
@@ -28,6 +28,7 @@ import com.daml.lf.command
 import com.daml.lf.data.Ref._
 import com.daml.lf.data.{Ref, Time}
 import com.daml.lf.engine.script.{Converter, LfValueCodec}
+import com.daml.lf.language.Ast
 import com.daml.lf.language.Ast._
```

Comment:
nipicking: prefix or not prefix ?

Reply:
The `import Ast._` predated my change. But I changed to use `Ast.` in all places.